src/snowflake/snowpark/functions.py

```diff
@@ -79,19 +79,7 @@
     ---------------------------
     <BLANKLINE>
 
-A user-defined function (UDF) can be called by its name with :func:`call_udf`:
```

Comment:
There is no need to advertise `call_udf` any more since `call_function` and `function` work for all functions including system-builtin and UDFs.